### PR TITLE
add unic framework

### PR DESCRIPTION
**Pull request by rajkumardusad**
Add unic framework benchmark

### CODE REVIEW

**Comment by rajkumardusad** [comment]
Hey @waghanza how can it solve this error?
Error: Cannot perform an interactive login from a non TTY device

**Comment by waghanza** [comment]
> Hey @waghanza how can it solve this error?
> Error: Cannot perform an interactive login from a non TTY device

Seems to be an error with CI (secret sharing between forks), I will try to solve it this week end

**Comment by rajkumardusad** [comment]
Ok

**Comment by waghanza** [comment]
@rajkumardusad does `unic` support php 8 ?

**Comment by rajkumardusad** [comment]
> @rajkumardusad does `unic` support php 8 ?

Yes

**Comment by rajkumardusad** [comment]
> > Hey @waghanza how can it solve this error?
> > Error: Cannot perform an interactive login from a non TTY device
> 
> Seems to be an error with CI (secret sharing between forks), I will try to solve it this week end

This issue is solved.

**Review by waghanza — CHANGES_REQUESTED** [review]
CI is OK 🎉 
https://github.com/the-benchmarker/web-frameworks/pull/3909/checks?check_run_id=1843260184

However, could you remove unused file (`README`, `.env`, ...), then idea is to have only required file (clearer)

File: php/unic/.env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Database Configuration
```
[the remark this answers is not in the document]
Comment (rajkumardusad) [reply php/unic/.env:1]: No

File: php/unic/.htaccess [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# REDIRECT ALL REQUEST TO INDEX.PHP
```
[the remark this answers is not in the document]
Comment (rajkumardusad) [reply php/unic/.htaccess:1]: No

**Comment by rajkumardusad** [comment]
> CI is OK 🎉
> https://github.com/the-benchmarker/web-frameworks/pull/3909/checks?check_run_id=1843260184
> 
> However, could you remove unused file (`README`, `.env`, ...), then idea is to have only required file (clearer)

Yes we have removed unused files.

File: php/unic/application/template/errors/403.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```
Comment (waghanza) [inline php/unic/application/template/errors/403.html:1]: is it not an unused file @rajkumardusad ?

Comment (rajkumardusad) [reply php/unic/application/template/errors/403.html]: Its error template like 404, 500 etc.
But we can remove it.

**Comment by rajkumardusad** [comment]
All files removed

**Comment by rajkumardusad** [comment]
I have tested on local it works, when i push there are some issue. i don't know how cant i solve it?

**Comment by waghanza** [comment]
I think the only thing left is that `index.php` is not in the right place, should be in a directory named `public` @rajkumardusad

**Comment by rajkumardusad** [comment]
But previously i have pushed same code and it passed all tests.

**Comment by waghanza** [comment]
> But previously i have pushed same code and it passed all tests.

indeed, the spec was not ... doing their job ...

**Comment by rajkumardusad** [comment]
> > But previously i have pushed same code and it passed all tests.
> 
> indeed, the spec was not ... doing their job ...

I don't know, but when update this PR with master branch then this issue arise.

**Comment by rajkumardusad** [comment]
Yes its done 😊

**Review by waghanza — APPROVED** [review]

**Pull request merged**